src/jit/hwintrinsicxarch.cpp

```diff
@@ -608,7 +608,7 @@ GenTree* Compiler::impSSEIntrinsic(NamedIntrinsic        intrinsic,
     GenTree* op3      = nullptr;
     GenTree* op4      = nullptr;
     int      simdSize = simdSizeOfHWIntrinsic(intrinsic, sig);
-    assert(simdSize == 16);
+    assert((simdSize == 16) || (simdSize == 0));
```

Comment:
This could use a comment about why `simdSize` would be zero (since the code for those is not really visible here).

Reply:
fixed.